tests/tests.sh

```diff
@@ -158,15 +165,15 @@ function compare_to_result()
 	local sav="$2"
 	if [ ! -r "$sav" ]; then return 0; fi
 	test "`cat -- "$sav"`" = "UNDEFINED" && return 1
-	diff -u -- "$sav" "$outdir/$pattern.res" > "$outdir/$pattern.diff.sav.log"
+	diff -u --strip-trailing-cr -- "$sav" "$outdir/$pattern.res" > "$outdir/$pattern.diff.sav.log"
```

Comment:
Sometimes (on Windows in particular), we want to actually test if the output contains `"\r"` or not (thinking of `"t"` mode versus `"b"` mode…)